libcipher/caesar.py

```diff
@@ -0,0 +1,64 @@
+__author__ = 'kskrishnasangeeth'
+
+import random
+import sys
+
+
+def encrypt(message, key=None):
+    """Returns the encrypted messsage for the string passed as argument.
+
+    Uses a random key for generating the cipher text
+
+    Args:
+        message (str) : The message to be encrypted.
+        key (int) : The key based on which encryption is performed. Is optional value.
+
+    Return:
+        str: the encrypted message """
```

Comment:
How I imagined you saying that: 

![56647084](https://cloud.githubusercontent.com/assets/240368/7204995/d815d7e4-e543-11e4-8033-5015025f82fa.jpg)